packages/transaction-preview/README.md

```diff
@@ -52,6 +53,102 @@ const onboard = Onboard({
 // The transaction will automatically be picked up and simulated with a UI displaying in the upper right corner
 ```
 
+### Standalone Usage
+
+To use the Transaction Preview package with web3-onboard all a developer needs to do is initialize web3-onboard with their [Blocknative API key](https://onboard.blocknative.com/docs/overview/introduction#optional-use-an-api-key-to-fetch-real-time-transaction-data-balances-gas) and pass in the module as shown below.
```

Comment:
I found this section a little confusing as the heading is about standalone usage, but the paragraph describes how to use with Onboard.

Reply:
Ah good catch!